kuma/health/tests/test_views.py

```diff
@@ -4,22 +4,27 @@
 from django.core.urlresolvers import reverse
 
 
-@pytest.mark.parametrize('http_method',
-                         ['get', 'head', 'put', 'post', 'delete', 'options'])
-def test_liveness(db, client, http_method):
+@pytest.mark.parametrize('http_method', ['put', 'post', 'delete', 'options'])
+@pytest.mark.parametrize('endpoint', ['health.liveness', 'health.readiness'])
```

Comment:
❤️ Nice little refactor to use stacked `pytest.mark.parametrize` decorators. I've got to remember to do this more often when it makes sense!